src/docfx/build/metadata/MetadataProvider.cs

```diff
@@ -41,7 +41,7 @@ public MetadataProvider(Docset docset, Cache cache, MicrosoftGraphCache microsof
             }
         }
 
-        public (List<Error> errors, InputMetadata metadata) GetMetadata(Document file)
+        public (List<Error> errors, InputMetadata metadata) GetInputMetadata(Document file)
```

Comment:
For `MetadataProvider`, should this still be `GetMetadata`, is there any other metadata?